cmd/internal/kube/print_snippet.go

```diff
@@ -0,0 +1,33 @@
+package kube
+
+import (
+	"github.com/postmanlabs/postman-insights-agent/cmd/internal/cmderr"
+	"github.com/postmanlabs/postman-insights-agent/printer"
+	"github.com/spf13/cobra"
+	"sigs.k8s.io/yaml"
+)
+
+var printHelmChartSnippetCmd = &cobra.Command{
+	Use:   "helm-sidecar-snippet",
+	Short: "Print a Helm chart container definition for adding the Postman Insights Agent to existing k8s deployment.",
+	Long:  "Print a container definition that can be inserted into a Helm Chart template to add the Postman Insights Agent as a sidecar container.",
+	RunE:  printHelmChartSnippet,
+}
+
+func printHelmChartSnippet(cmd *cobra.Command, args []string) error {
+	err := cmderr.CheckAPIKeyAndInsightsProjectID(insightsProjectID)
+	if err != nil {
+		return err
+	}
+
+	// Create the Postman Insights Agent sidecar container
+	container := createPostmanSidecar(insightsProjectID, false)
+
+	// Print the snippet
+	containerYaml, err := yaml.Marshal(container)
+	if err != nil {
+		return err
+	}
```

Comment:
Does this need to be indented further to be usable?  I am not sure how easy it will be to use.

Reply:
Done